packages/scandipwa/src/component/ProductReviews/ProductReviews.style.scss

```diff
@@ -88,7 +88,7 @@
             font-size: 18px;
         }
 
-        span {
+        span:nth-child(2) {
```

Comment:
Please use class names as they are more implicit selectors.
If someone changes the layout, this may break but remain unnoticed.